R/ergm.mple.R

```diff
@@ -140,8 +161,15 @@ ergm.mple<-function(nw, fd, m, init=NULL,
    }
   }
   real.coef <- mplefit$coef
+  
+  if(!is.dyad.independent(m) && control$init.method == "MPLE" && control$MPLE.covariance.method=="Godambe"){ 
+  real.cov <- mple.cov$Godambe
+  }else{ if(!is.dyad.independent(m) && control$init.method == "MPLE" && control$MPLE.covariance.method=="Bootstrap"){
+    real.cov <- mple.cov$Bootstrap  
+  }else{
   real.cov <- mplefit.summary$cov.unscaled
-
+  }}
+  
```

Comment:
Please try to use consistent indentation. Also, the code seems repetitive. Since "Godambe" and "bootstrap" are mutually exclusive, is there a reason they get different names in `mple.cov`?

Reply:
I tried to be more consistent with the indentation. There is also no reason for "Godambe" and "bootstrap" to get different names anymore. This is a remnant of my initial code. I have updated it.